pytket/qir/conversion/api.py

```diff
@@ -83,3 +85,33 @@ def pytket_to_qir(
         return populated_module.module.ir()
     else:
         raise ValueError("unsupported return type")
+
+
+def pytket_to_qir_optimised(
+    circ: Circuit,
+    name: str = "Generated from input pytket circuit",
+    returntype: ReturnTypeQIR = ReturnTypeQIR.BINARY,
+) -> Union[str, bytes, None]:
+    """converts given pytket circuit to qir
+    :param circ: given circuit
+    :type circ: pytket circuit
+    :param name: name for the qir module created
+    :type name: str
+    :param returntype: format of the generated qir, defaut value is binary
+    :type returntype: ReturnTypeQIR
+    """
+
+    initial_result = str(pytket_to_qir(circ, name, ReturnTypeQIR.STRING))  # type: ignore
+
+    initial_result = initial_result.replace("entry_point", "EntryPoint")  # type: ignore
+    initial_result = initial_result.replace("num_required_qubits", "requiredQubits")  # type: ignore
+    result = initial_result.replace("num_required_results", "requiredResults")  # type: ignore
```

Comment:
I tried this locally and now the submission fails with a completely unhelpful error message:

```
E               pytket.extensions.quantinuum.backends.quantinuum.GetResultFailed: Cannot retrieve result; job status is CircuitStatus(status=<StatusEnum.ERROR: 'Circuit has errored. Check CircuitStatus.message for error message.'>, message='{"name": "job", "submit-date": "2023-05-24T15:40:02.098446", "result-date": null, "queue-position": null, "cost": null, "error": null}', error_detail=None, completed_time=None, queued_time=None, submitted_time=None, running_time=None, cancelled_time=None, error_time=None, queue_position=None)
```

Reply:
Interesting, is that what we are expecting because of the missing register handling?

Reply:
I guess we would expect it to fail, yeah.